community/saas-companies-engagement-procedure.md

```diff
@@ -0,0 +1,118 @@
+## Procedure for Engaging With SaaS Companies
+
+### Summary
+
+This document outlines the procedure for engaging with SaaS companies, and
+how SaaS companies can represent themselves and their interests in the SPIFFE
+community.
+
+The goal is to provide a clear and consistent process for engaging with 
+SaaS companies that are included but not limited to those that use SPIFFE and 
+SPIRE, provide a proprietary SPIFFE-compatible service, or are interested in 
+contributing to the SPIFFE project.
+
+More specifically, this document includes guidelines for communication, 
+technical contact designation, and handling potential disruptive participants.
+
+### Technical Contacts
+
+#### Objective
+
+To ensure effective communication and support for SaaS consumers 
+using the SPIFFE specification by designating responsible technical contacts.
+
+#### Guidelines
+
+##### Designation of Contacts
+
+Each SaaS company that directs customers to the SPIFFE Slack channel must 
+designate two technical contacts.
+
+These contacts will serve as the primary point of contact for addressing queries 
+and facilitating communication between the customers and the Spiffy community.
+
+##### Documentation of Contacts
+
+A dedicated folder for each company will be created in the SPIFFE repository
+under the `./community/saas-technical-contacts` directory.
+
+The folder will contain:
+
+* Names and contact information of the designated technical contacts.
+* Guidelines for interaction and support procedures.
+* Any relevant documentation provided by the company.
+
+### Interaction Protocols
+
+#### Objective
+
+To establish clear protocols for interactions between SaaS consumers and the 
+SPIFFE community to ensure a productive and respectful environment.
+
+#### Guidelines
+
+##### Communication Etiquette
+
+All interactions should be professional, respectful, and focused on technical 
+issues.
+
+Technical contacts are responsible for ensuring that their customers, and anyone
```

Comment:
I appreciate the sentiment here, but it's a little unclear what the extent of this responsibility is. Whilst I wouldn't knowingly direct a customer who would act contrary to the guidelines towards this slack, in the case of generally linking to the SPIFFE community within documentation and such, it's difficult for a vendor to control/moderate the actions of those who follow a link in public documentation.